src/utils/Countries.ts

```diff
@@ -0,0 +1,116 @@
+// more countries @ https://github.com/umpirsky/country-list
+import countryData from 'country-data'
+import esData from 'src/country.json'
```

Comment:
I wonder if we could remove this specific one 🤔

Reply:
@jeanregisser Yeah, I'll look into it!